### PR TITLE
UDENG-133 Fixing some i18n problems

### PR DESCRIPTION
**Pull request by denisonbarbosa**
We had some issues related to the i18n package. This PR aims to solve them.

### CODE REVIEW

File: cmd/adwatchd/commands/run.go [hunk 1]
```diff
@@ -18,12 +18,8 @@ func (a *App) installRun() {
 	cmd := &cobra.Command{
 		Use:   "run",
 		Short: i18n.G("Starts the directory watch loop"),
-		Long: i18n.G(`Can run as a service through the service manager or interactively as a standalone application.
-
-The program will monitor the configured directories for changes and bump the appropriate GPT.ini versions anytime a change is detected.
-If a GPT.ini file does not exist for a directory, a warning will be issued and the file will be created. If the GPT.ini file is incompatible or malformed, the program will report an error.
-`),
-		Args: cobra.NoArgs,
+		Long:  i18n.G("Can run as a service through the service manager or interactively as a standalone application.\n\nThe program will monitor the configured directories for changes and bump the appropriate GPT.ini versions anytime a change is detected.\nIf a GPT.ini file does not exist for a directory, a warning will be issued and the file will be created. If the GPT.ini file is incompatible or malformed, the program will report an error.\n"),
```
Comment (CarlosNihelton) [inline cmd/adwatchd/commands/run.go:21]:
```suggestion
Long:  i18n.G("Can run as a service through the service manager or interactively as a standalone application.\n\nThe program will monitor the configured directories for changes and bump the appropriate GPT.ini versions anytime a change is detected.\nIf a GPT.ini file does not exist for a directory, a warning will be issued and the file will be created.\nIf the GPT.ini file is incompatible or malformed, the program will report an error.\n"),
```

The `If the GPT.init...` sentence lives in its own line.

Comment (CarlosNihelton) [reply cmd/adwatchd/commands/run.go:21]: Viewing the diff in the "files" tab made me think that there is:
image

File: cmd/adsysd/main.go [hunk 2]
```diff
@@ -22,6 +22,7 @@ import (
 
 func main() {
 	var a app
+	i18n.InitI18nDomain(consts.TEXTDOMAIN)
```
Comment (didrocks) [inline cmd/adsysd/main.go:25]: I think the a declaration is for the daemon below. So I would just move that one up. It’s not C89 :)

**Review by didrocks — CHANGES_REQUESTED** [review]
One requested change, and I think we should, before merging this, consider if we can’t search for an alternative on string extractions.

**Comment by EduardGomezEscandell** [comment]
> can we consider/research any other approach to extract the sentences in po files and to be compatible with bacquote?

Regarding extracting the strings: It is easy, I did it as a side thing here https://github.com/EduardGomezEscandell/go-gettext-adapter. I used this to generate PR https://github.com/ubuntu/adsys/pull/659.

The issue is maintaining this code (or a cleaner, better, not-done-in-one-afternoon version of it) and wether it is worth the hassle.

The linked repo already scans and finds the strings, it'd be a question of putting them in our potfile instead of sanitizing them.

**Comment by didrocks** [comment]
And with the new i18n, I think we can drop this one!

**Pull request closed without merging**